src/core/BufferAttribute.js

```diff
@@ -102,6 +102,16 @@ THREE.BufferAttribute.prototype = {
 
 	},
 
+	setRange: function ( index, typedArray ) {
```

Comment:
Decided against multiplying by itemSize in set , it reduces flexibility. Keeping parameter as offset rather than index to highlight the difference